website/markdown/docs/commands/migration.mdx

```diff
@@ -79,3 +79,26 @@ tuist migration check-empty-settings -p Project.xcodeproj -t MyApp
     },
   ]}
 />
+
+## List targets sorted by dependencies
+
+Migration of big Xcode projects to Tuist can happen iteratively, one target at a time. It makes sense to start from the target with the lowest number of dependencies.
+To help with that, Tuist includes a command that lists the targets of a project sorted by number dependencies ascending. 
+
+```bash
+tuist migration list-targets -p Project.xcodeproj
+```
```

Comment:
@andreacipriani I'd add a note saying that this only works for the dependencies defined through build phases. If they are defined implicitly using build settings the command will not include them. For example, CocoaPods dependencies are defined in `.xcconfig` files.

Reply:
Makes sense, updated!